src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -257,7 +257,7 @@ private void checkResult(final Map<String, JsonNode> expectedData) throws IOExce
             String id = (String) pair.getKey();
             expectedJSON = (JsonNode) pair.getValue();
 
-            long stopTime = System.currentTimeMillis() + 30000;
+            long stopTime = System.currentTimeMillis() + 120000;
```

Comment:
Why so long timeout?

Reply:
To compensate for the cases when Travis is slow and it takes long time to process events.

I will change to OR operator in the code so that if foundMatch is true we will not wait for stop time to expire like we do today.

Reply:
ok